todoman/interactive.py

```diff
@@ -62,10 +60,8 @@ def has_priority(self):
         '''
         Returns True iff the Item refers to a Todo with a priority that
         is set (i.e. not None and above zero).
-
-        (self) -> bool
         '''
-        return self.todo.priority not in [None, 0]
+        return self.todo.priority not in (None, 0)
```

Comment:
Maybe it's time to implement `Todo.is_urgent` as a wrapper around this, so we stop duplicating `todo.priority not in [None, 0]` al over the place.

Also, I think it can never be None, because of how the getter is written.